CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ If you use compression with a non-default collation on a segmentby-column you mi
 * #4120 Add logging for retention policy
 * #4169 Add support for chunk exclusion on DELETE to PG14
 * #4209 Add support for chunk exclusion on UPDATE to PG14
+* #4080 Add support for bulk inserts in COPY operator
```

Comment:
```suggestion
* #301 Add support for bulk inserts in COPY operator
```
we link the PR adding the feature here

Reply:
Good point. It is now changed to the PR.